vars/uiVeri5ExecuteTests.groovy

```diff
@@ -0,0 +1,152 @@
+ * * Full power of webdriverjs, protractor and jasmine - deferred selectors, custom matchers, custom locators.
+ * * Control locators (OPA5 declarative matchers) allow locating and interacting with UI5 controls.
+ * * Does not depend on testability support in applications - works with autorefreshing views, resizing elements, animated transitions.
+ * * Declarative authentications - authentication flow over OAuth2 providers, etc.
+ * * Console operation, CI ready, fully configurable, no need for java (comming soon) or IDE.
+ * * Covers full ui5 browser matrix - Chrome,Firefox,IE,Edge,Safari,iOS,Android.
+ * * Open-source, modify to suite your specific neeeds.
+ *
+ * !!! note "Browser Matrix"
+ *     With this step and the underlying Docker image ([selenium/standalone-chrome](https://github.com/SeleniumHQ/docker-selenium/tree/master/StandaloneChrome)) only Chrome tests are possible.
+ *
+ *     Testing of further browsers can be done with using a custom Docker image.
+ */
+@GenerateDocumentation
+void call(Map parameters = [:]) {
+    handlePipelineStepErrors (stepName: STEP_NAME, stepParameters: parameters) {
+        def script = checkScript(this, parameters) ?: this
+        def utils = parameters.juStabUtils ?: new Utils()
+
+        // load default & individual configuration
+        Map config = ConfigurationHelper.newInstance(this)
+            .loadStepDefaults()
+            .mixinGeneralConfig(script.commonPipelineEnvironment, GENERAL_CONFIG_KEYS)
+            .mixinStepConfig(script.commonPipelineEnvironment, STEP_CONFIG_KEYS)
+            .mixinStageConfig(script.commonPipelineEnvironment, parameters.stageName?:env.STAGE_NAME, STEP_CONFIG_KEYS)
+            .mixin(parameters, PARAMETER_KEYS)
+            .addIfEmpty('seleniumHost', isKubernetes()?'localhost':'selenium')
+            .use()
+
+        new Utils().pushToSWA([
+            step: STEP_NAME,
+            stepParamKey1: 'scriptMissing',
+            stepParam1: parameters?.script == null
+        ], config)
+
+        config.stashContent = config.testRepository ? [GitUtils.handleTestRepository(this, config)] : utils.unstashAll(config.stashContent)
+        config.installCommand = SimpleTemplateEngine.newInstance().createTemplate(config.installCommand).make([config: config]).toString()
+        config.runCommand = SimpleTemplateEngine.newInstance().createTemplate(config.runCommand).make([config: config]).toString()
+
+        seleniumExecuteTests(
+            script: script,
+            buildTool: 'npm',
+            dockerEnvVars: config.dockerEnvVars,
+            dockerImage: config.dockerImage,
+            dockerName: config.dockerName,
+            dockerWorkspace: config.dockerWorkspace,
+            sidecarEnvVars: config.sidecarEnvVars,
+            sidecarImage: config.sidecarImage,
+            stashContent: config.stashContent
+        ) {
+            try {
+                sh "NPM_CONFIG_PREFIX=~/.npm-global ${config.installCommand}"
```

Comment:
just as an info for having the same knowledge on each side: We need a test which execute an npm command (something like `npmExecute` which is close to this here @yemengsap @1084565 .